.github/workflows/php.yml

```diff
@@ -10,36 +10,44 @@ jobs:
   build:
 
     runs-on: ubuntu-latest
+    continue-on-error: ${{ matrix.experimental }}
     strategy:
       fail-fast: false
       matrix:
-        php-versions: ['7.2', '7.3', '7.4']
-    name: Tests with PHP ${{ matrix.php-versions }}
+        php-version:
+          - "7.2"
+          - "7.3"
+          - "7.4"
+        dependencies:
+          - "lowest"
+          - "highest"
+        experimental:
+          - false
+        include:
+          - php-version: "8.0"
+            dependencies: "highest"
+            composer-options: "--ignore-platform-reqs"
+            experimental: true
```

Comment:
Shall we remove `experimental: true` in anticipation of landing #205?

Reply:
Yes, I will once it will be merged

Reply:
https://github.com/magento/magento-coding-standard/pull/205 is merged

Reply:
Removed. Please review again